[FIX] website_sale_qty: Product page permissions

obulkin opened the pull request:
* Fix permission errors on website shop product pages by granting portal and public users read access to the ``product.uom`` model

obulkin committed:
[FIX] website_sale_qty: Product page permissions
* Fix permission errors on website shop product pages by granting portal and
public users read access to the product.uom model

pedrobaeza commented on website_sale_price_tier/__manifest__.py:
```diff
@@ -5,7 +5,7 @@
     'name': 'Website Sale - Price Tiers',
     'summary': 'Add price tier radio buttons to product pages in website shop',
-    'version': '10.0.1.0.0',
+    'version': '10.0.1.0.1',
```
It should be 10.0.1.1.0, as it requires to update module.

obulkin replied:
A module update is required, but the change is backwards compatible and is just a bugfix rather than a new feature, so changing the patch version seemed like the best option. That being said, I'm happy to use ``10.0.1.1.0`` instead.

pedrobaeza commented on website_sale_price_tier/security/ir.model.access.csv:
```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+product_uom_portal_read_access,Product UoM - Portal Read Access,product.model_product_uom,base.group_portal,1,0,0,0
```
Why do you need portal access? This is for website visitors only.

obulkin replied:
I commented on this in the v9 PR.

obulkin committed:
PR commit

obulkin commented:
@pedrobaeza - I've updated the version number as requested and modified the tour to not use a login so that permissions issues like this one can be better detected in the future.

yajo reviewed (approved):
code ok
